go/ct/spc/specification_test.go

```diff
@@ -101,10 +101,14 @@ func TestSpecification_EachRuleProducesAMatchingTestCase(t *testing.T) {
 	}
 }
 
-func TestSpecification_SstoreWithTooLittleGasProducesMatchingTestCases(t *testing.T) {
+// the following test has been used to verify stability of a specific rule.
+// so far it has been used for:
+// - sstore_with_too_little_gas_
+// - pc_on_data_is_ignored
```

Comment:
nit: This is code comment not documentation and should be placed inside the function